cyanobyte/codegen.py

```diff
@@ -33,11 +33,14 @@
 # Use the module title to import correctly in a Pip bundle
 from cyanobyte.convert_json_to_yaml import convert_json_to_yaml
 
+# via Python 3.7+, use this in Pip bundle to access correct resource filepath
+import importlib.resources as pkg_resources
+
 _VERSION = "0.1.0"
 _DEBUG = False
 _CLEAN = False
 _TEMPLATES = dict(
-    arduino=["./templates/arduino.cpp", "./templates/arduino.h"],
+    arduino=["arduino.cpp", "arduino.h"],
```

Comment:
Need to update all of these links to remove the './templates'.

What do we do in the case that we're developing this? Do we care as much since we can just point to the files directly?

Make sure this change won't break CI.